Activities/MultiplayerGame.cpp

```diff
@@ -327,9 +327,10 @@ namespace RTE {
 							saveSettings = true;
 						}
 
-						if (saveSettings)
-							g_SettingsMan.Save(Writer("Base.rte/Settings.ini"));
-
+						if (saveSettings){
```

Comment:
Might be nice to replace this with the slightly more compact SettingsMan::UpdateSettings.

Reply:
indeed

Reply:
There was another case of this below in the same file.